qiskit/assembler/assemble_schedules.py

```diff
@@ -92,9 +93,18 @@ def _assemble_experiments(
     instruction_converter = instruction_converter(qobj.PulseQobjInstruction,
                                                   **run_config.to_dict())
 
-    schedules = [
-        sched if isinstance(sched, pulse.Schedule) else pulse.Schedule(sched) for sched in schedules
```

Comment:
This was a bit tricky to figure out, also because in the mypi signature it reads:
`schedules: List[pulse.Schedule]`

I suppose, also looking at tests, that should better be `List[pulse.ScheduleComponent]`?

Is it possible to have other types, like (time, instruction) tuples?

Reply:
Yes, it is also possible to get a `(time, instruction)` tuple, this conversion would have been handled by the `pulse.Schedule` in the past.